frontend/src/components/AcmDataForm.tsx

```diff
@@ -124,6 +124,7 @@ export function AcmDataFormPage(props: AcmDataFormProps): JSX.Element {
 
   const { editorTitle, schema, secrets, immutables, formData, globalWizardAlert, hideYaml, isModalWizard } = props
   const [showFormErrors, setShowFormErrors] = useState(false)
+  const showErrors = props.mode === 'wizard' ? showFormErrors : formData.showErrors ?? false
```

Comment:
I don't think this is correct. `formData.showErrors` will only ever be true for your one use-case with Ansible credential validation. The idea is to force the showing of form errors when this happens, but the existing mechanism that uses `showFormErrors` still needs to work.